portal/plugins/gnmatomresponder/master_importer.py

```diff
@@ -179,6 +179,23 @@ def import_new_item(self, master_item, content, parent=None):
         from pac_xml import PacXmlProcessor
         from mock import MagicMock
         if not isinstance(master_item, VSItem) and not isinstance(master_item, MagicMock): raise TypeError #for intellij
+        from portal.plugins.kinesisresponder.sentry import inform_sentry_exception
+
+        vs_item_id = master_item.get("itemId")
+
+        try:
+            importjob = ImportJob.objects.get(item_id=vs_item_id)
+        except:
+            pass
+        else:
+            if importjob.processing == True:
+                logger.info('Data for item {0} already being processed. Aborting.'.format(vs_item_id))
+                inform_sentry_exception({
```

Comment:
there's no exception in context here so it probably won't work :(

`inform_sentry_exception` calls `raven.captureException` internally

You'll want `raven.captureMessage`  instead - probably add a new method to   `portal.plugins.kinesisresponder.sentry` to call this.

Reply:
Luckily it already has such a method so I will attempt to use it.